plugin.video.alphatv.gr/addon.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.alphatv.gr" version="1.0.0" name="ALPHA Player" provider-name="lambda">
+	<requires>
+		<import addon="xbmc.python" version="2.24.0" />
+		<import addon="script.module.lamlib" version="1.0.0" />
+		<import addon="plugin.video.youtube" optional="true" />
```

Comment:
The old VEVO add-on had the same "optional" for YouTube. No idea if it's valid or not, doesn't seem to hurt anything.